docs/en/operations/system-tables/zookeeper_log.md

```diff
@@ -0,0 +1,132 @@
+# system.zookeeper_log {#system-zookeeper_log}
+
+This table contains information about the parameters of the request to the ZooKeeper client and the response from it.
+
+For requests, only columns with request parameters are filled in, and the remaining columns are filled with default values (`0` or `NULL`). When the response arrives, the data from the response is added to the other columns.
+
+!!! info "Note"
+    The table does not exist if ZooKeeper is not configured.
+
+Columns with request parameters:
+
+-   `type` ([Enum](../../sql-reference/data-types/enum.md)) — Event type in the ZooKeeper client. Can have one of the following values:
+    -   `request` — The request has been sent.
+    -   `response` — The response was received.
+    -   `finalize` — The connection is lost, no response was received.
+-   `event_date` ([Date](../../sql-reference/data-types/date.md)) — The date when the request was completed.
```

Comment:
There are 3 types of events. Request is completed only when response is received (i.e. when a row with `type='Responce'` is written into the table). 
```suggestion
-   `event_date` ([Date](../../sql-reference/data-types/date.md)) — The date when the event happened.
```